src/sidebar/components/test/annotation-header-test.js

```diff
@@ -27,18 +27,20 @@ describe('sidebar.components.annotation-header', function() {
   });
 
   beforeEach('Import and register the annotationHeader component', function() {
-    const annotationHeader = proxyquire('../annotation-header', {
+    angular.module('app', []).component('annotationHeader', annotationHeader);
```

Comment:
Nit-Seems like we should be mocking the imports before attaching the component just in terms of flow for the reader but functionally it doesn't matter.

Reply:
You're right that it makes no functional difference, but I've switched it around anyway.